### PR TITLE
webmd.com #23671

### PR DESCRIPTION
**Pull request by slavaleleka**
https://github.com/AdguardTeam/AdguardFilters/issues/23671

### CODE REVIEW

**Review by BlazDT — CHANGES_REQUESTED** [review]
Move #$# rules to general_extensions.txt. 
Cookie message not reproduced with existing value.

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -507,6 +507,7 @@ thewpclub.net##.style1 > .note2_sp
 thewpclub.net##.widget_better-ads
 thewpclub.net##.bsac
 webmd.com##div.ad[id^="ads"]
+||sdad.guru/i10c@p1^$domain=webmd.com,third-party
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt]: better try this `||g00.webmd.com^$script`

Comment (slavaleleka) [reply EnglishFilter/sections/specific.txt]: it is hiding too much
here

Comment (Alex-302) [reply EnglishFilter/sections/specific.txt]: Ok. But this is InstartLogic ads reinjection script. Please try JS rules from Eng filter.

**Comment by slavaleleka** [comment]
social filter updated as well — social buttons are not hidden by browser extension in chrome

**Comment by BlazDT** [comment]
Why genericblock and generichide?

**Comment by slavaleleka** [comment]
@Alex-302 recommended me to try those

**Review by Alex-302 — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**